templates/setup.sh

```diff
@@ -1,9 +1,11 @@
 #!/usr/bin/env bash
 
-bundle install
 <% if enable_settings && dependency_manager_enabled?("cocoapods") %>
 pod install
 <% end %>
 <% if dependency_manager_enabled?("carthage") %>
 carthage update
 <% end %>
+<% if dependency_manager_enabled?("bundler") %>
+bundle install
```

Comment:
Probably it makes sense to do it first, so CocoaPods can be installed via bundler?